src/NuGet.Clients/NuGet.CommandLine/Commands/Pack/AssemblyMetadataExtractor.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using NuGet.Runtime;
+
+namespace NuGet.CommandLine
+{
+    using NuGet.Packaging;
+    using Versioning;
```

Comment:
The Versioning using statement could be changed to NuGet.Versioning and put with the others, but the NuGet.Packaging using statement needed to be inside the namespace otherwise there was confusion about whether to use the NuGet.PackageBuilder from NuGet.Core.dll(v2) or the NuGet.Packaging.PackageBuilder from NuGet.Packaging.dll